Dockerfile.release

```diff
@@ -9,7 +9,7 @@ COPY blade /usr/local/bin/
 EXPOSE 8545 9632 1478
 
 RUN addgroup -S blade-group \
-    && adduser -S blade -G blade-group
+    && adduser -S -u 1001 blade -G blade-group
```

Comment:
Just for me to understand, what do we need user id for?

Reply:
We are encountering an issue with the new network deployment, which involves using Docker images. We need to change the `blade` user ID to 1001 to establish it properly.